samples/word/05-scenarios/content-generation.yaml

```diff
@@ -0,0 +1,507 @@
+order: 1
+id: word-scenarios-content-generation
+name: Content Generation
+description: Insert predefined or AI-generated content to different locations of the document.
+host: WORD
+api_set:
+    WordApi: '1.5'
+script:
+    content: |
+        $("#generate").click(() => tryCatch(generateTemplate));
+        $("#skip-generate").click(() => tryCatch(updatePageDisplay));
+        $("#back").click(() => tryCatch(back));
+        $("#title-ai").click(() => tryCatch(insertAIGenerateTitle));
+        $("#title-predefined").click(() => tryCatch(insertPredefinedTitle));
+        $("#comment-ai").click(() => tryCatch(insertAIGenerateComment));
+        $("#comment-predefined").click(() => tryCatch(insertPredefinedComment));
+        $("#citation-ai").click(() => tryCatch(insertAIGenerateCitation));
+        $("#citation-predefined").click(() => tryCatch(insertPredefinedCitation));
+        $("#format").click(() => tryCatch(formatDocument));
+
+        let azureAIParams = {
+          endpoint: "",
+          deployment: "",
+          apiKey: "",
+          apiVersion: "2023-05-15"
+        };
+
+        const azureAIPrompts = {
+          title: "generate a title of meeting notes",
+          comment: "generate a comment of meeting notes",
+          citation: "generate a citation of meeting notes"
+        };
+
+        const predefinedData = {
+          title: "This is a predefined title",
+          comment: "This is a predefined comment",
+          citation:
+            "The Power of Storytelling in Education.\" TED, www.ted.com/talks/amanda_stevens_graham_the_power_of_storytelling_in_education."
+        };
+
+        function checkazureAIParams(): boolean {
```

Comment:
These pop-ups get cutoff in the task pane. Is there a way to prevent that?

![image](https://github.com/OfficeDev/office-js-snippets/assets/38896772/d0bd4bc7-8f2c-4341-8a72-16a54610a907)